### DISCUSSION TITLE
BAD_GATEWAY : ROUTER_CANNOT_MATCH

### DISCUSSION THREAD

**Discussion by Gmulti** [vercel/next.js#48168] [Help] [answered]
### Summary

Hello,

I am migrating Next 12 to Next 13, no problem locally and on a local build. 
I also tested with `vercel dev` and everything is ok.

At the time of putting in prod, I find myself with this message: 
<img width="561" alt="Capture d’écran 2023-04-09 à 22 10 14" src="https://user-images.githubusercontent.com/5682535/230794338-14ab844d-3aeb-4cff-8eb3-e38a7ecbd4f0.png">

I've tried a lot of manipulations but nothing works, even though everything is fine locally.

Do you have an idea of the problem?

Regards

**Comment by Gmulti** [comment]
After several searches, it seems to be caused with the 13.3 because I have no problems with 13.2

**Comment by samuelcole** [comment]
i just had the same experience coming from `13.2.4`

``` diff
+ "next": "^13.3.0",
- "next": "^13.2.4",
```

![image](https://user-images.githubusercontent.com/18301/231294069-34c0e3d5-9d20-4547-b920-bbf10ad27dad.png)

pretty alarming, would not recommend

**Comment by ijjk** [comment, marked as answer]
Hi, this should be fixed in the latest canary of Next.js `v13.3.1-canary.0`, please update and give it a try!

**Comment by Valid** [comment]
![image](https://github.com/vercel/next.js/assets/261192/3d27f0fa-5bde-466e-97bb-57992a886b57)
Still getting this with `v13.4.19`

I haven't seen any information about what could be causing this. No indication from build logs, and things work perfectly on local
